packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -3529,10 +3529,8 @@ export class ThinEngine extends AbstractEngine {
         }
     }
 
-    protected _deleteTexture(texture: Nullable<WebGLTexture>): void {
-        if (texture) {
-            this._gl.deleteTexture(texture);
-        }
+    protected _deleteTexture(texture: Nullable<WebGLHardwareTexture>): void {

```

Comment:
Please make sure to change NativeEngine as well. CC @bghgary

Reply:
Just a thought - it seems like this change makes _deleteTexture redundant, which makes sense to me (though I will wait for other reviewers). It is possible to remove `_deleteTexture`, and only use the texture's release function.

Reply:
Indeed, it looks like _deleteTexture can be deleted. 

I haven't worked with the NativeEngine, and I can't tell if I can delete the `_deleteTexture` from there as well. 
`INativeEngine` says the function `deleteTexture` is there, which is used by the `nativeEngine`'s `_deleteTexture`. But I can't find any implementers of the function anywhere. Thus I can't say if it's safe to just remove `_deleteTexture` from the `nativeEngine`.
I'm also unsure what type it expects. The nativeEngine's version takes a WebGLTexture, but uses it as an Uint32Array.

Reply:
let's keep it this way for now. I'll wait for the native team to comment here.